code/modules/power/hydroelectric.dm

```diff
@@ -179,6 +179,7 @@
 				var/malftrigger = rand(1,workingturbines)
 				connected_turbines[malftrigger].Malfunction()
 				malfnumber = 0
+			tidechange = 0
```

Comment:
If this is a boolean, please use FALSE.

Reply:
It's not a boolean, it increments to 100 before ticking over and triggerign that. Previously I forgot to set it to reset back to 0, so it would only run once per round before just going on its merry way to kingdom come.

Reply:
Got it.